spec/services/search/twitter_service_spec.rb

```diff
@@ -21,7 +21,9 @@
   end
 
   describe '.make_a_item_by' do
-    let(:tweet) { Twitter::Tweet.new(id: '596008529309278208', text: 'a text tweet') }
+    let(:tweet) { Twitter::Tweet.new(id: '596008529309278208', text: 'a text tweet',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.